lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -776,8 +783,9 @@ def _validate_single_linetime_pixelsize(self):
             )
         )
 
-        if line_times_err_msg or px_size_err_msg:
-            raise ValueError(" ".join([line_times_err_msg, px_size_err_msg]))
+        is_valid = not (line_times_err_msg or px_size_err_msg)
+        err_msg = "" if is_valid else " ".join([line_times_err_msg, px_size_err_msg])
+        return is_valid, err_msg
```

Comment:
nitpick (feel free to ignore): This seems a bit C-style to me. Not a big deal, but could just throw and catch it if you can handle it?

Reply:
That's how it was originally but then it seemed to get messy in `ensemble_diffusion`; but I think I can live with this solution